featuretools/tests/primitive_tests/test_postal_primitives.py

```diff
@@ -0,0 +1,44 @@
+import pandas as pd
+from woodwork.accessor_utils import _is_dask_series, _is_spark_series
+
+from featuretools.primitives.standard.transform.postal import (
+    OneDigitPostalCode,
+    TwoDigitPostalCode,
+)
+
+
+def generate_actual_and_expected_for_postal_primitive(postal_series, prim):
+    prim_ops = {
+        "one_digit_postal_code": lambda t: str(t)[0] if pd.notna(t) else pd.NA,
+        "two_digit_postal_code": lambda t: str(t)[:2] if pd.notna(t) else pd.NA,
+    }
+    op = prim_ops[prim.name]
+    actual = prim().get_function()(postal_series)
+    if _is_spark_series(postal_series):
+        expected = postal_series.apply(op).to_pandas()
+        actual = actual.to_pandas()
+    else:
+        expected = postal_series.apply(op)
+
+    if _is_dask_series(postal_series):
+        actual = actual.compute()
+        expected = expected.compute()
+    return actual, expected
```

Comment:
Sorry if it feels like I'm being picky here, but I think I would use this function to only get the expected values. Call the primitive function directly in the test to get the actual values.

Also, I think we should remove the `.compute()` and `to_pandas()` calls in this function and use our testing utility function `to_pandas` in `es_utils.py` to handle all of the conversions to pandas.

If we remove those calls and use `to_pandas()` instead, do we even need to retain this function? Maybe not since most of the logic here seems to be handling conversion to pandas.